tests/unit/_internal/service_loader/bento_with_models/service.py

```diff
@@ -0,0 +1,9 @@
+import bentoml
+
+
+test_model = bentoml.models.get("testmodel")
+# get by alias
+another_model = bentoml.models.get("another")
+svc = bentoml.Service(
+    "test-bento-service-with-models", models=[test_model, another_model]
+)
```

Comment:
Do we need another service file that explicitly have no model at all? Then when we load the bento, we should still have one model because the modelspec in bentofile.yaml

Reply:
No, this is only for testing service loading. The said testing point should be covered by https://github.com/bentoml/BentoML/blob/6d0839fe54572270f3842597e827a422b268a2c1/tests/unit/_internal/bento/test_bento.py#L321-L336

Reply:
got it